scripts/ci/appveyor/install.ps1

```diff
@@ -1,5 +1,5 @@
 function install() {
-    conda config --set auto_update_conda off
+    conda config --set auto_update_conda on
```

Comment:
@bryevdv, this "fixed" issues on Appveyor, but I'm not sure if this the way to go, and if so, should we change this on travis ci as well.

Reply:
I would prefer to lock down the version of conda used. We have upgraded to >=4.7 on TravisCI. I don't think I did anything to update the Appveyor config though (not sure where it is specified offhand) I would say we could first try updating Appveyor to use the latest conda version, and keep auto-update off